src/abstractdataframe/reshape.jl

```diff
@@ -211,6 +212,21 @@ function unstack(df::AbstractDataFrame, colkey::Int, value::Int)
     unstack(df2, length(df2), colkey, value)
 end
 
+import Base.convert
+
+function convert{T}(vector::AbstractVector{T})
```

Comment:
Better to call this some other function; here you're defining `convert` on a `AbstractVector` which could lead to weirdness in people's other code that's totally unrelated.